nilearn/_utils/glm.py

```diff
@@ -134,7 +134,9 @@ def coerce_to_dict(input_arg):
     return input_arg
 
 
-def make_stat_maps(model, contrasts, output_type="z_score"):
+def make_stat_maps(
```

Comment:
The naming should make it explicit that this is computing second-level contrasts on the inputs: first-level models and corresponding contrasts.

Reply:
I'd like that but this function is supposed to work with both first and second level models, so it may be better to keep generic.